packages/components/src/item-group/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Item, ItemGroup } from '..';
+//  import { CONFIG } from '../../utils';
+
+describe( 'ItemGroup', () => {
+	describe( 'ItemGroup component', () => {
+		it( 'should render correctly', () => {
+			const wrapper = render(
+				<ItemGroup>
+					<Item>Code is poetry</Item>
+				</ItemGroup>
+			);
+			expect( wrapper ).toMatchSnapshot();
```

Comment:
This should be `wrapper.container.firstChild`
```suggestion
			expect( wrapper.container.firstChild ).toMatchSnapshot();
```

Reply:
Done in [`1374e2a` (#33701)](https://github.com/WordPress/gutenberg/pull/33701/commits/1374e2a051f1c157079666ce569acb3278f9f2a9)